marimo/_plugins/ui/_impl/table.py

```diff
@@ -246,20 +246,33 @@ def __init__(
                 "https://github.com/marimo-team/marimo/issues"
             )
 
-        row_limit = _internal_row_limit or TableManager.DEFAULT_ROW_LIMIT
+        if _internal_row_limit is not None:
+            self._row_limit = _internal_row_limit
+        else:
+            self._row_limit = TableManager.DEFAULT_ROW_LIMIT
```

Comment:
This rewrite is to keep the code style consistent with `if _internal_total_rows is None` below.